### PR TITLE
Docs (mostly): remove caskroom/unofficial

### PR DESCRIPTION
**Pull request by vitorgalvao**

### CODE REVIEW

File: doc/development/adding_a_cask.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Forks should have the vendor’s name as a prefix on the Cask’s file name and
 
 ### Unofficial, Vendorless, and Walled Builds
 
-Please submit these to [caskroom/homebrew-unofficial](http://github.com/caskroom/homebrew-unofficial). If you’ve made an unofficial build and need a place to host it, contact our sister project [alehouse](https://github.com/alehouse).
+We do not accept these casks since they offer a higher-than-normal security risk. [alehouse/homebrew-unofficial](https://github.com/alehouse/homebrew-unofficial) is a sister repo where you may wish to submit your cask.
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: @alebcay Will https://github.com/alehouse/homebrew-unofficial be the correct repo? Any objection to this information?

Comment (alebcay) [inline doc/development/adding_a_cask.md]: Looks fine to me.

File: USAGE.md [hunk 2]
```diff
@@ -145,9 +145,9 @@ The primary Homebrew-Cask Tap includes most of the Casks that a typical user wil
 
 | Tap name | description |
 | -------- | ----------- |
-| [caskroom/versions](https://github.com/caskroom/homebrew-versions)     | contains alternate versions of Casks (e.g. betas, nightly releases, old versions)
-| [caskroom/fonts](https://github.com/caskroom/homebrew-fonts)           | contains Casks that install fonts, which are kept separate so we can educate users about the different licensing landscape around font installation/usage
-| [caskroom/unofficial](https://github.com/caskroom/homebrew-unofficial) | contains Casks that install unofficial builds or forks
+| [caskroom/versions](https://github.com/caskroom/homebrew-versions) | contains alternate versions of Casks (e.g. betas, nightly releases, old versions)
+| [caskroom/fonts](https://github.com/caskroom/homebrew-fonts)       | contains Casks that install fonts, which are kept separate so we can educate users about the different licensing landscape around font installation/usage
+| [caskroom/ied](https://github.com/caskroom/homebrew-ied)           | contains Casks that install electronic identity card software of various countries
```
Comment (tsparber) [inline USAGE.md]: ied -> eid :smiley:

Comment (vitorgalvao) [inline USAGE.md]: Thank you.

**Pull request merged**